Sources/PackageLoading/ManifestLoader+Validation.swift

```diff
@@ -107,18 +107,9 @@ public struct ManifestValidator {
                 continue
             }
 
-            for (index, unicodeScalar) in traitName.unicodeScalars.enumerated() {
-                let properties = unicodeScalar.properties
-
-                if index == 0 {
-                    if !(properties.isIDStart || properties.isASCIIHexDigit || unicodeScalar == "_") {
-                        diagnostics.append(.invalidFirstCharacterInTrait(firstCharater: unicodeScalar, trait: trait.name))
-                    }
-                } else {
-                    if !(properties.isXIDContinue || unicodeScalar == "_" || unicodeScalar == "+") {
-                        diagnostics.append(.invalidCharacterInTrait(character: unicodeScalar, trait: trait.name))
-                    }
-                }
+            guard traitName.isValidIdentifier else {
```

Comment:
suggestion : Can we add `small` (ie: unit tests) tests that validates this function in isolation?